### DISCUSSION TITLE
Is it possible to update file permissions in /var/lib/kubelet/pods?

### DISCUSSION THREAD

**Discussion by harsimranmaan** [k3s-io/k3s#10731] [Q&A]
The files are created with 666+ permissions
eg
 -rw-rw-rw- root root /var/lib/kubelet/pods/19530219-2c1b-4a87-8355-92c66c75fd49/containers/prometheus-server-configmap-reload/39cb132e
 
 or 
 drwxrwsrwx root abc /var/lib/kubelet/pods/e85daa92-92bd-4f01-8f8b-be812d628774/volumes/kubernetes.io~empty-dir/tmp
 
 
 Is there a way to restrict file permission to avoid world read/write? Looked at k8s upstream an kubelet config options but did not find anything so far. Similar to https://github.com/kubernetes/kubernetes/issues/80872

**Comment by brandond** [comment]
This is a Kubernetes question, not RKE2. Our Kubelet is the vanilla Kubernetes kubelet.
